lib/core/src/math/utils.rs

```diff
@@ -0,0 +1,9 @@
+use nalgebra::clamp;
+use std::ops::Mul;
+use std::ops::Add;
+
+pub fn linear_interpolation<T>(a:T, b:T, t:f32) -> T
+    where T : Mul<f32, Output = T> + Add<T, Output = T> {
+    let t = clamp(t, 0.0, 1.0);
+    a * (1.0 - t) + b * t
+}
```

Comment:
This thing is used in physics only. I don't like to create top-level "math" module now as we don have anything to put there. If we will have more similar elements, lets do it. For now, please keep this function as an util inside of hte physics engine.

The idea here is simple - let's keep in modules similar things. If we have to create a top-level module for a single function, there is probably something broken with our design. I dont know yet how it will evolve in the future, but for now, lets keep the top-level module system minimal :)

Reply:
@wdanilo I have plans to use it in `animators/EasingAnimator` and in Slider UI element, so I think it's a good idea get this function available and accessible in `math`.